mlir/lib/Dialect/SparseTensor/Transforms/SparseTensorPasses.cpp

```diff
@@ -260,6 +260,7 @@ struct SparseTensorCodegenPass
     // The following operations and dialects may be introduced by the
     // codegen rules, and are therefore marked as legal.
     target.addLegalOp<linalg::FillOp>();
+    target.addLegalOp<linalg::YieldOp>();
```

Comment:
Is a test failing if you don't add that?

Reply:
yep, without this legalizing `YieldOp`, the `RewritePattern` won't be applied (since YieldOp it's not legal).